src/aap_eda/api/views/activation.py

```diff
@@ -278,7 +271,30 @@ def disable(self, request, pk):
 
         if activation.is_enabled:
             activation.is_enabled = False
-            activation.save(update_fields=["is_enabled", "modified_at"])
+
+            job_id = activation.current_job_id
+
+            if job_id:
+                queue = get_queue()
+                job = queue.fetch_job(job_id)
+                if job and job.get_status() in [
+                    "queued",
```

Comment:
I feel all these states should be defined in a common dataclass and be consumed from there to avoid using arbitrary hardcoded strings. What do you think?